.github/workflows/perf_micro.yml

```diff
@@ -72,6 +76,25 @@ jobs:
           revision: ${{ inputs.revision }}
       - name: test
         run: make -f .test.mk test-perf
+      - name: Send statistics to InfluxDB
+        # TODO: For now, use the debug bucket for this PoC.
+        # --silent -o /dev/null: Prevent dumping any reply part
+        # in the output in case of an error.
+        # --fail: Exit with the 22 error code is status >= 400.
+        # -w: See the reason for the failure, if any.
```

Comment:
Long option is  `--write-out`.

Reply:
Thanks! Renamed to the long name of the option.

Reply:
Thanks!